### ISSUE TITLE
Courses/Tracks don't show

### ISSUE THREAD

**Issue by Bcopeland64** [TRoboto/datacamp-downloader#41]
First off...great tool. However, when I login, I can't seem to get a list of courses or completed tracks listed. The list doesn't populate when I run 'datacamp courses', nor will it list completed tracks when I run 'datacamp tracks'.  Is my syntax wrong? Did I miss something?

**Comment by TRoboto** [comment]
Have you completed the courses/tracks you want to download? The tool only lists completed courses/tracks.

**Comment by Bcopeland64** [comment]
Yes...in fact, I have looked exclusively at courses I have finished, and I can see the frame of where the courses would be, but they don't populate.I used 'datacamp download all' and 'datacamp download all -t' I can send screenshots if necessary.Sent from my Galaxy
-------- Original message --------From: Mohammad Al-Fetyani ***@***.***> Date: 8/12/21  4:14 PM  (GMT+02:00) To: TRoboto/datacamp-downloader ***@***.***> Cc: Brandon Copeland ***@***.***>, Author ***@***.***> Subject: Re: [TRoboto/datacamp-downloader] Courses/Tracks don't show (#41) 
Have you completed the courses/tracks you want to download? The tool only lists completed courses/tracks.

**Comment by Bcopeland64** [comment]
Thanks for your help...great tool you created!Sent from my Galaxy
-------- Original message --------From: Mohammad Al-Fetyani ***@***.***> Date: 8/12/21  4:47 PM  (GMT+02:00) To: TRoboto/datacamp-downloader ***@***.***> Cc: Brandon Copeland ***@***.***>, Author ***@***.***> Subject: Re: [TRoboto/datacamp-downloader] Courses/Tracks don't show (#41) 
LOL, the website just changed the html source code of the profile page. I will have to implement the new changes. Probably will take a day or so, stay tuned!

**Comment by TRoboto** [comment]
This issue is now resolved in PR #42,  but not yet merged. Please confirm that it's indeed fixed. 

For now, install the new version with:
```
pip install -U git+https://github.com/TRoboto/datacamp-downloader.git@fixbug
```

**Comment by ElipLam** [comment]
> This issue is now resolved in PR #42, but not yet merged. Please confirm that it's indeed fixed.
> 
> For now, install the new version with:
> 
> ```
> pip install -U git+https://github.com/TRoboto/datacamp-downloader.git@fixbug
> ```

This work for me, great tool!
Thank you so much.

**Comment by TRoboto** [comment]
> This work for me, great tool!
> Thank you so much.

Thanks for the confirmation! Closing this issue.
